public_html/assets/js/nf-core-schema-builder.js

```diff
@@ -695,22 +788,23 @@ function generate_param_row(id, param){
 
     var results = `
     <div class="row schema_row border" data-id="`+id+`">
-        <div class="col-sm-auto align-self-center d-none d-sm-block schema_row_grabber border-right">
+        <div class="col-auto align-self-center schema_row_grabber border-right">
             <i class="fas fa-grip-vertical"></i>
         </div>
-        <div class="col-sm-auto align-self-center d-none d-sm-block param_fa_icon ">`+fa_icon+`</div>
+        <button class="col-auto align-self-center param_fa_icon ">`+fa_icon+`</button>
         <div class="col schema-id">
             <label>ID
                 <input type="text" class="text-monospace param_id" value="`+id+`">
             </label>
         </div>
-        <div class="col-sm-3">
+        <div class="d-sm-none w-100"></div>
+        <div class="col">
             <label>Description
                 <input type="text" class="param_key param_description" data-param_key="description" value="`+param['description']+`">
             </label>
         </div>
-        <div class="col-sm-auto align-self-center schema_row_help_text_icon">`+help_text_icon+`</div>
-        <div class="col-sm-1">
+        <div class="col-auto align-self-center schema_row_help_text_icon iconpicker-component">`+help_text_icon+`</div>
```

Comment:
Why is this class needed here?

Reply:
Yeah, it wasn't! I noticed it as well, now removed. 👍